typing/typecore.ml

```diff
@@ -744,6 +750,40 @@ let add_pattern_variables ?check ?check_as env pv =
     )
     pv env
 
+let add_module_variables env unpacks ~outer_level =
+  List.fold_left (fun env unpack ->
+    Typetexp.TyVarEnv.with_local_scope begin fun () ->
+      (* This code is parallel to the typing of Pexp_letmodule. However we
+         omit the call to [Mtype.lower_nongen] as it's not necessary here.
+         For Pexp_letmodule, the call to [type_module] is done in a raised
+         level and so needs to be modified to have the correct, outer level.
+         Here, on the other hand, we're calling [type_module] outside the
+         raised level, so there's no extra step to take.
+      *)
+      let modl, md_shape =
+        !type_module env
+          Ast_helper.(
+            Mod.unpack ~loc:unpack.tu_loc
+              (Exp.ident ~loc:unpack.tu_name.loc
+                  (mkloc (Longident.Lident unpack.tu_name.txt)
+                    unpack.tu_name.loc)))
+      in
+      Mtype.lower_nongen outer_level modl.mod_type;
```

Comment:
This has crept back in, presumably accidentally.

Reply:
Yes, a mistake. It's gone now.